simplesam.py

```diff
@@ -262,8 +263,8 @@ def _header_dict_format(self):
 
     def write(self, sam):
         """ Write the string representation of the ``sam`` :class:`.Sam` object. """
-        self.file.write(str(sam))
-        
+        print(str(sam), end="", file=self.file)
```

Comment:
Do you have a reason for using the `print` function here as opposed to calling the write method of `self.file`? I've just tested that your example using `sys.stdout` works just fine with the existing code. If there's a good reason for this change I'm interested, otherwise could you please revert the changes to `simplesam.py`? If you'd like I can just copy your contribution to the Readme and give you attribution in the commit message.

Reply:
Short answer, no: I am so used to `print` that I didn't realize `write` can do the same :sweat_smile: 
What's the best way to revert? I don't want to mess up with commits and pull requests.